src/ducks/sharing/index.js

```diff
@@ -38,6 +38,44 @@ export const createPermSet = (id, doctype) =>
     }
   })
 
+export const share = async (id, doctype, email, url) => {
+  const target = await cozy.client.fetchJSON('POST', '/sharings/recipient', {
+    email,
+    url
+  })
+
+  const sharings = await cozy.client.fetchJSON('POST', '/sharings/', {
+    desc: 'Share Ilyn Sarkisian',
```

Comment:
https://fr.wikipedia.org/wiki/Cher_(artiste) :')
But also yeah maybe put the name of the album there or something^^